cardano_node_tests/tests/test_dbsync.py

```diff
@@ -229,11 +229,14 @@ def test_cost_model(self, cluster: clusterlib.ClusterLib):
         protocol_params = cluster.g_query.get_protocol_params()
         pp_cost_models = protocol_params["costModels"]
 
+        # TODO: remove `PlutusScriptV1` old option after new release
```

Comment:
Better to be more specific so we have enough details once we decide to drop support for old versions:

```python
# TODO: `PlutusScriptV1` was replaced with `PlutusV1` in node version 8.0.0
```